### PR TITLE
Update collection metada - Web

### PR DESCRIPTION
**Pull request by jakzaizzat**
### Summary of Changes

- Added trigger point to the community page (will open the request changes form)
- Added formspree detail tot the vercel env for (preview/dev/prod)

### Demo or Before/After Pics

https://github.com/gallery-so/gallery/assets/4480258/ed59163b-ccdd-4864-b9d9-998887723f04

https://github.com/gallery-so/gallery/assets/4480258/6c304ff6-924e-4ca2-9ba4-70de699b95f0

### Edge Cases

List any common edge cases that you have considered and tested.

### Testing Steps

Provide steps on how the reviewer can test the changes.

### Checklist

Please make sure to review and check all of the following:

- [x] I've tested the changes and all tests pass.
- [x] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.

### CODE REVIEW

File: apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        `https://formspree.io/f/${process.env.NEXT_PUBLIC_FORMSPEE_REQUEST_COLLECTION_ID}`,
+        {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            communityId: community.dbid,
+            communityName: community.name,
+            message,
```
Comment (kaitoo1) [inline apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx:52]: nice!! could you also include the submitter's user dbid? 🙏

File: apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx [hunk 2]
```diff
@@ -185,7 +212,12 @@ export default function CommunityPageViewHeader({ communityRef, queryRef }: Prop
     const buttonText = showExpandedDescription ? 'Show Less' : 'Show More';
 
     return (
-      <StyledDescriptionWrapper gap={8}>
+      <StyledDescriptionWrapper
+        gap={8}
+        onClick={handleEditClick}
```
Comment (kaitoo1) [inline apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx:217]: I noticed the editor opens if I click on "Show More" or "Show Less" on the description. could we block that event so it doesn't trigger the editor if I'm toggling the description expansion? should be the same for clicking on links in the description too.

CleanShot 2024-02-02 at 21 10 17

File: apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx [hunk 3]
```diff
@@ -198,18 +230,50 @@ export default function CommunityPageViewHeader({ communityRef, queryRef }: Prop
             onClick={handleShowMoreClick}
           />
         )}
+
+        <StyledEditIconWrapper>
+          <EditPencilIcon />
+        </StyledEditIconWrapper>
+
+        <StyledTooltip
```
Comment (kaitoo1) [inline apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx:238]: I think we only want to show the tooltip if you hover on the pencil icon

CleanShot 2024-02-02 at 21 13 18@2x
CleanShot 2024-02-02 at 21 13 14@2x

Comment (kaitoo1) [reply apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx:238]: so it looks like it should be:

hover on editable area: show pencil icon
hover on pencil icon: show tooltip

Comment (kaitoo1) [reply apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx:238]: lets make the tooltip appear on the right of the icon too, right now it's on top

oh also for the icon, we have a `IconContainer` component that maybe you can try - it has the tooltip built in, and also has a `tooltipPlacement` prop that lets you specify which direction it appears in!

File: apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+        )}
+      </VStack>
+      <ButtonContainer>
+        <Button
+          eventElementId="Community Metadata Form Submit Button"
+          eventName="Submit Community Metadata Form"
+          eventContext={contexts.Community}
+          onClick={handleSubmit}
+        >
+          Submit
```
Comment (kaitoo1) [inline apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx:98]: oo could you disable this button if there's no input? (ie disabled by default, becomes enabled when the user types)

File: apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx (hunk 1 above)
Comment (jakzaizzat) [reply apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx:70]: Added userId!

File: apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx (hunk 4 above)
Comment (jakzaizzat) [reply apps/web/src/scenes/CommunityPage/CommunityMetadataFormModal.tsx:118]: Updated!

File: apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx (hunk 3 above)
Comment (jakzaizzat) [reply apps/web/src/scenes/CommunityPage/CommunityPageViewHeader.tsx]: Thanks for tips! Refactor it to use the `IconContainer` and its working great. The code much cleaner too now 🫠


https://github.com/gallery-so/gallery/assets/4480258/0b283dfa-c1e3-4778-ba40-4b6070311b3e

**Review by kaitoo1 — APPROVED** [review]

**Pull request merged**